app/assets/stylesheets/admin/rebate_form.css

```diff
@@ -227,8 +227,14 @@
 
 .rebate-qrcode {
   display: flex;
-  margin: 1em auto;
+  margin: 5em auto;
   background-color: black;
+  width: 20%;
```

Comment:
Are these lines just for centering it on the page?
```
 position: absolute;
  top: 50%;
  left: 50%;
  margin-right: -50%;
  transform: translate(-50%, -50%);
```

Reply:
Yup :stuck_out_tongue: